widgets/accordion/styles/default.less

```diff
@@ -64,20 +70,33 @@
 			}
 
 			.sow-accordion-title {
-				display: inline-block;
-				width: ~"calc(100% - 25px)";
+				display: flex;
+				align-items: center;
+				& when ( @heading_title_align = center ) {
+					justify-content: center;
+				}
+				
+				& when ( @heading_title_align = right ) {
+					justify-content: end;
+				}
+
+				width: 100%;
 				& when ( @show_open_close_icon = true ) and ( @heading_title_align = @open_close_location ) {
 					margin-@{open_close_location}: 5px;
 				}
 			}
 
-			.sow-accordion-open-close-button {
-				float: @open_close_location;
-
-				& when ( @show_open_close_icon = false ) {
-					display: none;
+			& when ( @show_open_close_icon = false ) {
+				.sow-accordion-open-close-button {
+						display: none;
 				}
 			}
+
+			.sow-accordion-icon-image {
```

Comment:
I know there is a big mix of formatting around but if possible, let's organize selector alphabetically when adding new rules. Thanks.